sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md

```diff
@@ -2,6 +2,10 @@
 
 ## Current version
 ###  Feature Additions
+- Added support for Data Flow:
+        * Added data flow object model and CRUD APIs
```

Comment:
nit: Keep whitespaces the same, copy paste from other lines

Reply:
I need the intend in order to add these as sub list